modules/preview-react/form-field/lib/FormFieldLabel.tsx

```diff
@@ -1,103 +1,81 @@
 import React from 'react';
 
-import {createSubcomponent} from '@workday/canvas-kit-react/common';
-import {useFormFieldLabel, useFormFieldModel} from './hooks';
-import {createStencil, px2rem} from '@workday/canvas-kit-styling';
-import {mergeStyles} from '@workday/canvas-kit-react/layout';
+import {createSubcomponent, ExtractProps} from '@workday/canvas-kit-react/common';
+import {createStencil, parentModifier, px2rem} from '@workday/canvas-kit-styling';
+import {Text, textStencil} from '@workday/canvas-kit-react/text';
+import {FlexProps, mergeStyles} from '@workday/canvas-kit-react/layout';
 import {brand, system} from '@workday/canvas-tokens-web';
 
-export interface FormFieldLabelProps {
+import {useFormFieldLabel, useFormFieldModel} from './hooks';
+import {formFieldStencil} from './formFieldStencil';
+
+export interface FormFieldLabelProps
+  extends FlexProps,
+    Omit<ExtractProps<typeof Text, never>, 'display'> {
   /**
    * The text of the label.
    */
   children: React.ReactNode;
-  /**
-   * Will style the text as disabled
-   */
-  disabled?: boolean;
-  /**
-   * Changes the color of the text
-   */
-  variant?: 'error' | 'hint' | 'inverse';
 }
 
-const formFieldLabelAsteriskStencil = createStencil({
-  base: {
-    fontSize: system.fontSize.body.large,
-    fontWeight: system.fontWeight.normal,
-    color: brand.error.base,
-    textDecoration: 'unset',
-    marginInlineStart: system.space.x1,
-  },
-});
-
-const formFieldLabelStencil = createStencil({
+export const formFieldLabelStencil = createStencil({
+  extends: textStencil,
+  // @ts-ignore Still weird about CSS variables
   base: {
-    ...system.type.subtext.large,
     fontWeight: system.fontWeight.medium,
     color: system.color.text.default,
     paddingInlineStart: system.space.zero,
     marginBottom: system.space.x1,
     display: 'flex',
     alignItems: 'center',
     minWidth: px2rem(180),
-  },
-  modifiers: {
-    orientation: {
-      horizontal: {
-        float: 'left',
-        maxHeight: system.space.x10,
-      },
-      vertical: {
-        width: '100%',
-      },
+
+    '& :where([data-element=asterisk])': {
+      display: 'none',
```

Comment:
if it's display none, do you care about styling it? Can't you remove the styles below?